### DISCUSSION TITLE
Обход блокировки других сайтов помимо Ютуба.

### DISCUSSION THREAD

**Discussion by 345sdfasaszz** [bol-van/zapret#649] [Q&A]
Привет! Немного не по теме, но хотелось бы поинтересоваться, можно ли сделать точно такую же схему, но с обходом блокировки каких-либо других сайтов, например, Твиттера? Я не очень разбираюсь, как это работает, и единственное, что знаю, это то, что ваши файлы используют API самого Ютуба, а также, как я понимаю, меняют IP-адреса? Хотелось бы услышать ваше мнение, и, если не трудно, дать совет как можно это сделать самому. Спасибо большое!

**Comment by Letton** [comment]
Все ответы в документации. То что вы пишите - бред

**Reply by bol-van** [reply]
IP не меняется, API не используется. Как это работает описано в самом начале доки
